OneDrive-Cloud-Player/ViewModels/MainPageViewModel.cs

```diff
@@ -27,6 +27,7 @@ public class MainPageViewModel : ViewModelBase
         public ICommand LogoutCommand { get; set; }
         public ICommand ToParentFolderCommand { get; set; }
         public ICommand GetProfileInfoCommand { get; set; }
+        public ICommand NavigateToSettingsPageCommand { get; set; }
```

Comment:
Maybe a better name would be `ToSettingsPageCommand` to keep the naming consistent with the other commands.

Reply:
I use the `NavigateTo` to let the reader know a navigation will happen. But it can be changed to `To` as it also contains the word `Page`.